backend/app/services/doctransform/job.py

```
@@ -0,0 +1,272 @@
+from starlette.datastructures import Headers
+
+from app.crud.document.doc_transformation_job import DocTransformationJobCrud
+from app.crud.document.document import DocumentCrud
+from app.models import (
+    Document,
+    DocTransformJobUpdate,
+    TransformationStatus,
+    DocTransformationJobPublic,
+    TransformedDocumentPublic,
+    DocTransformationJob,
+    TransformedDocumentPublic,
```

Ruff (0.14.5)

Redefinition of unused `TransformedDocumentPublic` from line 20

Remove definition: `TransformedDocumentPublic`

(F811)